com.unity.netcode.gameobjects/Runtime/Serialization/BytePacker.cs

```diff
@@ -282,231 +282,183 @@ public static void WriteValuePacked(FastBufferWriter writer, string s)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void WriteValueBitPacked<T>(FastBufferWriter writer, T value) where T: unmanaged => writer.WriteValueSafe(value);
 #else
-
         /// <summary>
-        /// Maximum serializable value for a BitPacked ushort (minimum for unsigned is 0)
+        /// Obsolete value that no longer carries meaning. Do not use.
```

Comment:
Not sure if this is required but we might use this:
[ObsoleteAttribute("Obsolete value that no longer carries meaning. Do not use.", false)]
Which will generate warnings to the user.  
This also means that this PR requires a minor version update which I think the PR title should be prefixed with "feat!:"?

Reply:
Never mind... can't use ObsoleteAttribute on const values so disregard.